quartz/plugins/transformers/description.ts

```diff
@@ -19,19 +26,36 @@ export const Description: QuartzTransformerPlugin<Partial<Options> | undefined>
       return [
         () => {
           return async (tree: HTMLRoot, file) => {
-            const frontMatterDescription = file.data.frontmatter?.description
-            const text = escapeHTML(toString(tree))
+            let frontMatterDescription = file.data.frontmatter?.description
+            let text = escapeHTML(toString(tree))
+
+            if (opts.replaceExternalLinks) {
+              frontMatterDescription = frontMatterDescription?.replace(urlRegex, "$<domain>")
+              text = text.replace(urlRegex, "$<domain>")
+            }
 
             const desc = frontMatterDescription ?? text
-            const sentences = desc.replace(/\s+/g, " ").split(".")
+            const sentences = desc.replace(/\s+/g, " ").split(/\.\s/)
             let finalDesc = ""
             let sentenceIdx = 0
             const len = opts.descriptionLength
-            while (finalDesc.length < len) {
-              const sentence = sentences[sentenceIdx]
-              if (!sentence) break
-              finalDesc += sentence + "."
-              sentenceIdx++
+
+            if (sentences[0].length >= len) {
```

Comment:
`sentences[0]` can be `undefined`

Reply:
If have added a check for `sentences[0] !== undefined`.

I was wondering if it could ever be undefined. Even an empty markdown file seems to build succesfully, albeit with an empty string as description in the resulting html.